File: ios/Approach/Sources/StatisticsLibrary/Protocols/SecondRollStatistic.swift
import Charts
import ModelsLibrary
import StringsLibrary

public protocol SecondRollStatistic: PercentageStatistic, TrackablePerSecondRoll {}

extension SecondRollStatistic {
	public static var numeratorTitle: String { title }
	public static var includeNumeratorInFormattedValue: Bool { true }

	public var isEmpty: Bool {
		numerator == 0
	}
}

File: android/core/statistics/src/main/java/ca/josephroque/bowlingcompanion/core/statistics/interfaces/SecondRollStatistic.kt
package ca.josephroque.bowlingcompanion.core.statistics.interfaces

import ca.josephroque.bowlingcompanion.core.statistics.TrackablePerSecondRoll

interface SecondRollStatistic : PercentageStatistic, TrackablePerSecondRoll {
	override val numeratorTitleResourceId: Int
		get() = id.titleResourceId

	override val includeNumeratorInFormattedValue: Boolean
		get() = true

	override val isEmpty: Boolean
		get() = numerator == 0
}
